### PR TITLE
Made equal_splits use the first split_column if more than one is give rather than complain

### PR DESCRIPTION
**Pull request by erzel**
@dumbunny

---
This change is [Reviewable](https://reviewable.io/reviews/youtube/vitess/1651)

### CODE REVIEW

File: proto/vtgate.proto [hunk 1]
```diff
@@ -549,17 +549,20 @@ message SplitQueryRequest {
...
-  //    This is fast, but requires that the distribution of values of
-  //    split_column be uniform in [min, max], for the number of rows
-  //    returned by each query part to be roughly the same.
+  //    If this algorithm is selected then only the first split_column given is
+  //    used (or the first primary key column if no split_column's are
+  //    specified). In the rest of this algorithm's description, we refer to
+  //    this column as "the split column".
+  //    The split_column must have numeric type (integral or floating point).
+  //    The algorithm works by taking the interval [min, max], where min and
+  //    max are the minimum and maximum values of of the split_column in
```
Comment (dumbunny) [inline proto/vtgate.proto]: Get rid of "of of".

File: proto/vtgate.proto [hunk 2]
```diff
@@ -549,17 +549,20 @@ message SplitQueryRequest {
...
+  //    the table-shard, respectively, and partitioning it into split_count
+  //    sub-intervals of equal size. The added WHERE clause of each query-part
+  //    restricts that part to rows whose value in the split_column belongs to
```
Comment (dumbunny) [inline proto/vtgate.proto]: You are mixing underscore (split_column) and hyphen (sub-intervals, table-shard). You only need one way to group a multi-word term.

File: proto/vtgate.proto [hunk 3]
```diff
@@ -549,17 +549,20 @@ message SplitQueryRequest {
...
+  //    a particular sub-interval. This is fast, but requires that the
+  //    distribution of values of the split_column be uniform in [min, max],
```
Comment (dumbunny) [inline proto/vtgate.proto]: Omit trailing comma.

**Comment by dumbunny** [comment]
LGTM after fixing comment.

**Comment by erzel** [comment]
Review status: 0 of 3 files reviewed at latest revision, 3 unresolved discussions.

---

*[proto/vtgate.proto, line 558 \[r1\]](https://reviewable.io:443/reviews/youtube/vitess/1651#-KGIiVfYdUXAOhAyVBfk-r1-558:-KGJeFkIH00TPTRmMAv_:513574686) ([raw file](https://github.com/youtube/vitess/blob/66ee68b37623a4acfcebc973082d1bfe6b2809a7/proto/vtgate.proto#L558)):*
Thanks. Done.

---

*[proto/vtgate.proto, line 561 \[r1\]](https://reviewable.io:443/reviews/youtube/vitess/1651#-KGIiVfYdUXAOhAyVBfk-r1-561:-KGJeXcpfCBeK4UaHYzZ:-1402841146) ([raw file](https://github.com/youtube/vitess/blob/66ee68b37623a4acfcebc973082d1bfe6b2809a7/proto/vtgate.proto#L561)):*
'split_column' is the name of a field in this message so I'm using '_'. The other cases are english words and so I'm grouping them with a hyphen. I've changed it slightly to make it a little cleaner. Let me know if you have further objections.

---

*[proto/vtgate.proto, line 563 \[r1\]](https://reviewable.io:443/reviews/youtube/vitess/1651#-KGIiVfYdUXAOhAyVBfk-r1-563:-KGJfuPrp_GxTmi1AYzv:-469593373) ([raw file](https://github.com/youtube/vitess/blob/66ee68b37623a4acfcebc973082d1bfe6b2809a7/proto/vtgate.proto#L563)):*
done

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/youtube/vitess/1651)*

**Comment by dumbunny** [comment]
LGTM

**Pull request merged**